MHR_Overlay.lua

```diff
@@ -392,7 +392,7 @@ function record_health(enemy)
 
     local monster = monster_table[enemy];
 
-    if not hp_entry then 
+    if not monster then

```

Comment:
This is the actual bug that this is fixing (All other changes are due to cleaning up whitespace)

Reply:
Oh thanks! Must have missed that!